tests/integration/pilot/piggyback_test.go

```diff
@@ -64,37 +62,5 @@ func TestPiggyback(t *testing.T) {
 				}
 				return nil
 			})
-
-			expectSubstrings := func(have string, wants ...string) error {
-				for _, want := range wants {
-					if !strings.Contains(have, want) {
-						return fmt.Errorf("substring %q not found; have %q", want, have)
-					}
-				}
-				return nil
-			}
-
-			// Test gRPC-based Tap Service using istioctl.
-			retry.UntilSuccessOrFail(t, func() error {
```

Comment:
the other integration test was a duplicate of the base `istioctl ps`, but are we loosing coverage in this one by removing this test?  is it no longer valid or is it covered somewhere else?

Reply:
I think some of this was discussed here: https://github.com/istio/istio/pull/46050#issuecomment-1661953633

if this is a loss of functionality - do we want to remove the test, or do we want to create an issue for tracking and use a `t.Skip(<link to GitHub issue>)` ?

Reply:
You are right, this will lead to less coverage I think. I think before doing `t.Skip` or creating an issue, I'll look into the code, and try to replace it with a new test method to test the gPRC-based Tap instead of `istioctl x ps`.

Reply:
This should be addressed by https://github.com/istio/istio/pull/47801.